packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -85,6 +92,19 @@ export function packageManagerExec(script: string, options: ExecSyncOptions) {
   return execCmd(PACKAGE_MANAGERS_CMD[getPackageManager()].exec, script, options);
 }
 
+/**
+ * Execute a binary command (npx / yarn exec) for a specific workspace
+ * @param workspace
+ * @param script
+ * @param options
+ */
+export function packageManagerWorkspaceExec(workspace: string, script: string, options: ExecSyncOptions) {
+  if (getPackageManager() === 'npm') {
```

Comment:
Could be moved out as function (as it is used twice)

Reply:
done